README.md

```diff
@@ -1 +1,346 @@
-#  Chart repo
+# OpenShift Helm Charts Repository
+
+OpenShift Helm Charts Repository is the canonical source of all the certified
```

Comment:
canonical source to single source of truth

Reply:
I used "canonical source" based on this README:
https://github.com/operator-framework/community-operators#about-this-repository
"single source of truth" is too technical and not really required in the first line of the document.